app/src/main/AndroidManifest.xml

```diff
@@ -17,26 +18,26 @@
         tools:replace="android:icon"
         tools:targetApi="31">
         <activity
-            android:name=".creditCard.RefreshPage"
+            android:name=".creditCardUI.refreshPageUI.RefreshPage"
             android:exported="false" />
         <activity
-            android:name=".creditCard.credit_home_page"
+            android:name=".creditCardUI.creditPage.Credit_home_page"
             android:exported="false" />
         <activity
-            android:name=".creditCard.error_page"
+            android:name=".creditCardUI.error.Error_page"
             android:exported="false" />
         <activity
-            android:name=".creditCard.procedding_page"
+            android:name=".creditCardUI.proceedingPageUI.Procedding_page"
             android:exported="false" />
         <activity
-            android:name=".creditCard.user_details"
```

Comment:
go for single activity multiple fragments

Reply:
will do in next commit